Symfony/CS/DocBlock/Tag.php

```diff
@@ -46,10 +46,9 @@ public function __construct($content)
     {
         preg_match_all('/@[a-zA-Z0-9_]+/', $content, $matches);
 
+        $this->name = 'other';
         if (isset($matches[0][0])) {
             $this->name = strtolower(ltrim($matches[0][0], '@'));
-        } else {
-            $this->name = 'other';
```

Comment:
If-elses-else tends to grow, so idea is to state default value before any complex checks and modifications.

Works well especially for arrays, when each branch can define own structure and defining default array before saves control flow in long-time prospect.

Nevertheless better to follow team's habitats here, I want just to show possibility.